### PR TITLE
add AUTHORS file

### PR DESCRIPTION
**Pull request by graeme-winter**
👍

### CODE REVIEW

**Comment by graeme-winter** [comment]
@rjgildea commit which I think is a _very good idea_ - we should also add similar wording to CONTRIBUTING as pytest have e.g. "Unless your change is a trivial or a documentation fix (e.g., a typo or reword of a small section) please add yourself to the AUTHORS file, in alphabetical order."

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Contributors include::
```
Comment (Anthchirp) [inline AUTHORS]:
```suggestion
Contributors include:
```

**Comment by rjgildea** [comment]
As used in projects such as [django](https://github.com/django/django/blob/master/AUTHORS) and [pytest](https://github.com/pytest-dev/pytest/blob/master/AUTHORS).

File: AUTHORS (hunk 1 above)
Comment (ndevenish) [reply AUTHORS]: Depends. I think it's like this in pytest because they include it in their ReST documentation directly, and that makes it a literal block

**Pull request merged**